services/policies/README.md

```diff
@@ -0,0 +1,70 @@
+# Policies Service
+
+The Policies Service provides a new grpc api which can be used to return whether a requested operation is allowed or not.
+Open Policy Agent is used to determine the set of rules of what is permitted and what is not.
+
+Policies are written in the [rego query language](https://www.openpolicyagent.org/docs/latest/policy-language/).
+
+The Policies Service consists of the following modules
+
+* Proxy Authorization (middleware)
+* Event Authorization (async post-processing)
+* GRPC API (can be used from other services too)
+
+### GRPC service
+
+This service can be used from any other internal service, it can also be used for example by third parties to find out if an action is allowed or not.
+This layer is already used by the proxy middleware.
+
+### Event service
+
+This layer is event-based and part of the asynchronous post-processing, since processing at this point is asynchronous, the operations there can also take longer and be more expensive,
+the bytes of a file can be examined here as an example.
+
+### Proxy Middleware
+
+The [ocis proxy](../proxy) already includes such a middleware which uses the [GRPC service](#grpc-service) to evaluate the policies by using a configurable query.
+Since the Proxy is in heavy use and every request is processed here, only simple decisions that can be processed quickly are recommended here, more complex queries such as file evaluation should be avoided urgently.
+
```

Comment:
```suggestion
## Modules

### GRPC Service

This service can be used from any other internal service. It can also be used for example by third parties to find out if an action is allowed or not. This layer is already used by the proxy middleware.

### Event Service

This layer is event-based and part of the postprocessing service. Since processing at this point is asynchronous, the operations can also take longer and be more expensive, like evaluating the bytes of a file.

### Proxy Middleware

The [ocis proxy](../proxy) already includes such a middleware which uses the [GRPC service](#grpc-service) to evaluate the policies by using a configurable query. Since the Proxy is in heavy use and every request is processed here, only simple and quick decisions should be evaluated. More complex queries such as file evaluation are strongly discouraged.

```
I am not clear about the GRPC Service as there is nothing that can be configured. This means, in which cases is the policies service returing results to it? How can the policies service be understood when it comes to GRPC? The intro text says `can be used from other services` but what is meant with that. This needs more clarification.

Reply:
not sure if we should document it, the GRPC service is really low level, most of our services can be used via GRPC, but to do so you have to read code.